cookbooks/bcpc/attributes/default.rb

```diff
@@ -36,6 +36,7 @@
 default['bcpc']['bootstrap']['interface'] = "eth0"
 default['bcpc']['bootstrap']['pxe_interface'] = "eth1"
 default['bcpc']['bootstrap']['server'] = "10.0.100.3"
+default['bcpc']['bootstrap']['vip'] = node['bcpc']['bootstrap']['server']
 default['bcpc']['bootstrap']['dhcp_range'] = "10.0.100.14 10.0.100.250"
```

Comment:
`node['bcpc']['bootstrap']['dhcp_range']` and `node['bcpc']['bootstrap']['dhcp_subnet']` should go away here and in the environment.